"--crds" crashes flux in 0.17.2

### Describe the bug

flux 0.17.2 cli crashes when using `--crds` parameter. 
It was working fine in previous versions.

### Steps to reproduce

```bash
flux create helmrelease cert-manager \
  --namespace="cert-manager" \
  --interval="1h" \
  --source="HelmRepository/jetstack.flux-system" \
  --chart="cert-manager" \
  --chart-version="v1.5.3" \
  --crds="CreateReplace" \
  --export
```

Output:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x195358d]

goroutine 1 [running]:
main.createHelmReleaseCmdRun(0x2e1ff40, 0xc0000aa900, 0x1, 0x8, 0x0, 0x0)
	/home/runner/work/flux2/flux2/cmd/flux/create_helmrelease.go:200 +0x34d
github.com/spf13/cobra.(*Command).execute(0x2e1ff40, 0xc0000aa880, 0x8, 0x8, 0x2e1ff40, 0xc0000aa880)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:852 +0x472
github.com/spf13/cobra.(*Command).ExecuteC(0x2e1dc40, 0x0, 0x4063c5, 0xc000112058)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:960 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:897
main.main()
	/home/runner/work/flux2/flux2/cmd/flux/main.go:137 +0x4f
```

### Expected behavior

Flux should generate the HelmRelease correctly when using `--crds` parameter.

### Screenshots and recordings

_No response_

### OS / Distro

Ubuntu 20.04, MacOS

### Flux version

0.17.2

### Flux check

N/A

### Git provider

_No response_

### Container Registry provider

_No response_

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

--- a/cmd/flux/create_helmrelease.go
+++ b/cmd/flux/create_helmrelease.go
@@ -187,9 +187,11 @@ func createHelmReleaseCmdRun(cmd *cobra.Command, args []string) error {
 	}
 
 	if helmReleaseArgs.createNamespace {
-		helmRelease.Spec.Install = &helmv2.Install{
-			CreateNamespace: helmReleaseArgs.createNamespace,
+		if helmRelease.Spec.Install == nil {
+			helmRelease.Spec.Install = &helmv2.Install{}
 		}
+
+		helmRelease.Spec.Install.CreateNamespace = helmReleaseArgs.createNamespace
 	}
 
 	if helmReleaseArgs.saName != "" {

--- a/cmd/flux/create_helmrelease.go
+++ b/cmd/flux/create_helmrelease.go
@@ -197,6 +197,10 @@ func createHelmReleaseCmdRun(cmd *cobra.Command, args []string) error {
 	}
 
 	if helmReleaseArgs.crds != "" {
+		if helmRelease.Spec.Install == nil {
+			helmRelease.Spec.Install = &helmv2.Install{}
+		}
+
 		helmRelease.Spec.Install.CRDs = helmv2.Create
 		helmRelease.Spec.Upgrade = &helmv2.Upgrade{CRDs: helmv2.CRDsPolicy(helmReleaseArgs.crds.String())}
 	}

